rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/ldap/LdapClientTest.java

```diff
@@ -47,12 +48,29 @@ class LdapClientTest {
   @Spy @InjectMocks LdapClient cut;
 
   @Test
-  void testLdap_searchGroupByDn() {
+  void should_CallLdapCorrectly_When_SearchGroupByDn() {
+    setUpEnvMock();
+    cut.init();
+    List<String> inputList =
+        List.of(
+            "cn=Developersgroup,ou=groups,o=taskanatest",
+            "cn=developersgroup,ou=groups,o=taskanatest");
 
+    cut.searchAccessIdByDn(inputList.get(0));
+    cut.searchAccessIdByDn(inputList.get(1));
+
+    // Instead of TestFactory we used verify(any(), times(2)) since the usage of TestFactory will
+    // need a Mockito.reset() which is not recommended
+    verify(ldapTemplate, times(2))
+        .lookup(eq("cn=developersgroup,ou=groups"), any(), any(LdapClient.DnContextMapper.class));
+  }
```

Comment:
What's the usecase for this test?
`should_LdapCallWithLowercase_When_SearchGroupByDn()` already covers that an uppercase Group gets converted to lowercase.

Reply:
I just forgot to delete should_LdapCallWithLowercase_When_SearchGroupByDn()
How about deleting `should_LdapCallWithLowercase_When_SearchGroupByDn()` and using `should_CallLdapCorrectly_When_SearchGroupByDn()` since it covers both cases: lowercase and uppercase.

Reply:
done.